example/use_cases/employer_university_flow/main.go

```diff
@@ -0,0 +1,177 @@
+
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/sirupsen/logrus"
+)
+
+// Set to debug mode here
+var debug = os.Getenv("DEBUG")
+
+type Mode string
+
+const (
+	DebugMode = "1"
+)
+
+// set mode for debugging
+// in bash:
+// export DEBUG=1
+func init() {
+	if debug == DebugMode {
+		println("Debug mode")
+		logrus.SetLevel(logrus.DebugLevel)
+	}
+}
+
+// In this example, we will
+// buile a simple example of a standard flow
+// between a student, a university, and an employer
+// 1. A student graduates from a university.
+// The university issues a VC to the student, saying they graduated
+// 2. The student will store it in a "wallet"
+// 3. An employer sends a request to verify that the student graduated
+// the university.
+func main() {
+	example.CustomWriter.Write("Starting University Flow")
+
+	// Wallet initialization
+	example.CustomWriter.Write("Initializing Student")
+	student, err := emp.NewEntity("Student", "key")
+	util.HandleExampleError(err, "failed to create student")
+
+	example.CustomWriter.Write("Initializing Employer")
+	employer, err := emp.NewEntity("Employer", "peer")
+	util.HandleExampleError(err, "failed to make employer identity")
+	verifier_did, err := employer.GetWallet().GetDID("main")
+	util.HandleExampleError(err, "failed to create employer")
+
+	example.CustomWriter.Write("Initializing University")
+	university, err := emp.NewEntity("University", "peer")
+	util.HandleExampleError(err, "failed to create university")
+	vcDID, err := university.GetWallet().GetDID("main")
+
+	util.HandleExampleError(err, "falied to initialize verifier")
+	example.CustomWriter.WriteNote(fmt.Sprintf("Initialized Verifier DID: %s and registered it", vcDID))
+	emp.TrustedEntities.Issuers[vcDID] = true
+
+	// Creates the VC
+	example.CustomWriter.Write("Example University Creates VC for Holder")
+	example.CustomWriter.WriteNote("DID is shared from holder")
+	holderDID, err := student.GetWallet().GetDID("main")
+	util.HandleExampleError(err, "failed to store did from university")
+
+	vc, err := emp.BuildExampleUniversityVC(vcDID, holderDID)
+	util.HandleExampleError(err, "failed to build vc")
+
+	// Send to user
+	example.CustomWriter.Write("Example University Sends VC to Holder")
+	student.GetWallet().AddCredentials(*vc)
+	msg := fmt.Sprintf("VC puts into wallet. Wallet size is now: %d", student.GetWallet().Size())
+	example.CustomWriter.WriteNote(msg)
+
+	example.CustomWriter.WriteNote(fmt.Sprintf("initialized verifier DID: %v", verifier_did))
+
+	// 	Presentation Request
+	example.CustomWriter.Write("Employer wants to verify student graduated from Example University. Sends a presentation request")
+	presentationData, err := emp.MakePresentationData("test-id", "id-1")
+	util.HandleExampleError(err, "failed to create pd")
+	if dat, err := json.Marshal(presentationData); err == nil {
+		logrus.Debugf("Presentation Data:\n%v", string(dat))
```

Comment:
if there's an error here, the program will continue?

Reply:
yea...the error would be contained to marshaling data. 1) It shouldn't ever really happen ( a failure to marshal ) but I see it as a design choice not a bug if it decides to not exit. There could be cases, where json marshaling fails for some reason. 

I'll change it to exit but wanted to note the above.